tests/Dockerfile

```diff
@@ -7,13 +7,13 @@ RUN pip install poetry
 COPY ./pyproject.toml ./poetry.lock* /python/app/
 
 # Generate the requirement file 
-ARG DEVFLAG=""
```

Comment:
Not sure if you meant to leave that blank here but it was reading as you expected.

Reply:
Yeah I left it blank to have PRD as default .... maybe the syntax is wrong but it's doable.
Actually looks like `ARG DEVFLAG` would work

Reply:
Okay i figured out what the issue was with this one